src/platform/device.gni

```diff
@@ -108,8 +108,8 @@ declare_args() {
 
   # Enable Subscription persistence / resumption for CI and supported platforms
   if (chip_device_platform == "darwin" || chip_device_platform == "linux" ||
-      chip_device_platform == "esp32" || chip_device_platform == "fake" ||
-      chip_device_platform == "efr32" || chip_device_platform == "SiWx917") {
+      chip_device_platform == "fake" || chip_device_platform == "efr32" ||
```

Comment:
Why is the default here changing?  Or is it not relevant in that `chip_persist_subscriptions` will always be set to some value on ESP32 now?

Reply:
Since the `chip_persist_subsciptions` will be set in platform args in the CMakerLists.txt file for ESP32 at last, so I think we can remove the platform check here.